docs/GM-Commands.md

```diff
@@ -1,461 +1,2291 @@
-GM commands can be entered mainly in 2 ways. Either by typing it directly into the world console window. In the world console window the leading dot (.) is not mandantory, but you can use it. The other way is using the gm command ingame in the chat console of the wow client. There, all commands must start with a leading dot, f.ex.: .gm on
+<p>GM commands can be entered mainly in 2 ways. Either by typing it directly into the world console window. In the world console window the leading dot (.) is not mandantory, but you can use it. The other way is using the gm command ingame in the chat console of the wow client. There, all commands must start with a leading dot, f.ex.: .gm on</p>
```

Comment:
no? <p> is not read on markdown

instead put `**`

Verify here https://github.com/azerothcore/wiki/blob/642c64eaf09eb5c1d31b4a570081285eea45eacf/docs/GM-Commands.md

Reply:
I made it html because the wiki page didnt like markdown tables, was shown like this

![image](https://user-images.githubusercontent.com/24550914/59162428-65222a00-8af1-11e9-942c-8fd015a26691.png)

:D

Reply:
yes, for tables it's okay but as you can see, the `<p>` doesn't work
should be fixed (and not approved xD)

Reply:
Ahh, I was following how other wiki pages here was made, that was whh I added the <p> 😂

Reply:
other pages are using the `<p>` ? (if so it's because they were converted automatically and there was a mistake)

Also my bad, I confused p and b, I thought you wanted to bold them.

Reply:
Nah, I used <strong> for bold

Reply:
`<strong>`